evaluate-qanary-system.py

```diff
@@ -411,15 +415,15 @@ def determine_if_custom_module_is_callable(logger, custom_module):
 
     try:
         custom_module.validate  # no call, just checking the name, TODO: improve
-        logger.info("Method '%s' found in module '%s'." %
+        logger.debug("Method '%s' found in module '%s'." %
                     (method_name, custom_module.__name__))
     except Exception as e:
         logger.error("Method '%s' NOT found in module '%s'." %
                      (method_name, custom_module.__name__))
         raise RuntimeError("Your custom module '%s' needs to contain a method '%s' " % (
             custom_module.__name__, method_name))
 
-    if len(inspect.getargspec(custom_module.validate).args) != 5:
+    if len(inspect.getfullargspec(custom_module.validate).args) != 5:
```

Comment:
i got an error with old solution when running with newest python version.

a quick google search recommended this method and it works fine running the test inside a docker container